service/application/src/main/java/com/solace/maas/ep/event/management/agent/subscriber/messageProcessors/ScanCommandMessageProcessor.java

```diff
@@ -72,4 +72,9 @@ public Class supportedClass() {
     public ScanCommandMessage castToMessageClass(Object message) {
         return (ScanCommandMessage) message;
     }
+
+    @Override
+    public void onFailure(Exception e, ScanCommandMessage message) {
+        log.debug("Requires implementation");
+    }
```

Comment:
We must implement this for Scan.

Reply:
That's coming up next? Can we potentially hit this after the PR is merged? Seems more like a warning at least.

Reply:
@CameronRushton already implemented in another PR following this one